apps/rush-lib/src/logic/taskRunner/TaskRunner.ts

```diff
@@ -2,26 +2,28 @@
 // See LICENSE in the project root for license information.
 
 import * as os from 'os';
-import { Interleaver } from '@rushstack/stream-collator';
+import * as colors from 'colors';
 import {
-  Terminal,
-  ConsoleTerminalProvider,
-  Colors,
-  IColorableSequence,
-  AlreadyReportedError
-} from '@rushstack/node-core-library';
+  StdioSummarizer,
+  TerminalWritable,
+  StdioWritable,
+  TerminalChunkKind,
+  CharMatcherTransform
+} from '@rushstack/terminal';
+import { StreamCollator, CollatedTerminal, CollatedWriter } from '@rushstack/stream-collator';
+import { AlreadyReportedError, NewlineKind, InternalError, Sort } from '@rushstack/node-core-library';
 
 import { Stopwatch } from '../../utilities/Stopwatch';
 import { Task } from './Task';
 import { TaskStatus } from './TaskStatus';
-import { TaskError } from './TaskError';
+import { IBuilderContext } from './BaseBuilder';
 
 export interface ITaskRunnerOptions {
   quietMode: boolean;
   parallelism: string | undefined;
   changedProjectsOnly: boolean;
   allowWarningsInSuccessfulBuild: boolean;
-  terminal?: Terminal;
+  destination?: TerminalWritable;
```

Comment:
"destination" is confusing in this context. It kinda sounds like this is where build output should go.

Reply:
agreed